### DISCUSSION TITLE
Digitizing Lines on GPS Recorded Points Snaps to Lat/Long Position Instead of Project CRS

### DISCUSSION THREAD

**Discussion by DrawQuick** [opengisch/QField#5546] [Bugs]
I have recorded a number of points using a Layer with a defined CRS(EPSG:9989). When the points are recorded a transformation to the project CRS occurs. If I want to digitize a series of lines to connect these points, the result is a line which snaps to the Latitude/Longitude position of the point rather than the transformed position of the point. I have upload a project file under Discussion #5540.
To replicate the issue
1. Select a line type such as Group 31R1792F/StreetLine/lines
2. Enter manual digitize mode
3. Place digitizing cursor so it snaps to a blue point. You will notice it will disappear.
4. Press the Record button.
5. Move the digitize cursor to another blue point. The cursor will disappear again
6. Press the Record button.
7. Move the digitize cursor to a position which does not snap to a blue point and you will see the line drawing from the Latitude/Longitude position to current cursor position(Project CRS location).
The issue appears to be that no transformation of the GPS points to the project CRS is occurring when snapping.

**Comment by nirvn** [comment]
@DrawQuick , fixed here: https://github.com/opengisch/QField/pull/5547 -- if you can install the test APK to confirm this issue is fixed, it'd be great.

**Comment by DrawQuick** [comment]
Downloaded and tested. Works!
Thanks
